### DISCUSSION TITLE
Plugin or Connection with Google Tasks

### DISCUSSION THREAD

**Discussion by abdullahBizz** [Flow-Launcher/Flow.Launcher#2156] [Plugin Request]
Hey Guys, I have installed Flow after a long time and there is a significant improvement. But I think there is something missing. While browsing through the plugin store, I saw a plugin for Todoist, which is great! But I use Google Tasks as my main to-do app. There are several other apps like Google Tasks and Microsft To-Do. I was wondering if anyone can add Google Tasks to the plugin, it would be a lovely addition to have!

**Comment by deefrawley** [comment]
This is a plugin request so will move to discussions (doubt it will be added to the Todoist plugin as this was ported from Wox but there may be a plugin author who would like to do a Google Tasks one)
